packages/markdown/tests/plugins/importCodePlugin.spec.ts

```diff
@@ -282,22 +282,4 @@ foo
       expect(mockConsoleError).toHaveBeenCalledTimes(1)
     })
   })
-
-  describe('compatibility with codePlugin', () => {
```

Comment:
Should check compatibility with vPrePlugin then? Or do you have any plan to migrate importCodePlugin, too?

Reply:
currently, I think no, we can just leave it here, as it's not coupled with highlighter.

Reply:
> Should check compatibility with vPrePlugin then?

@pengzhanbo I think this should be needed